code/src/index.css

```diff
@@ -11,3 +11,115 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+.questions {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  color: black;
+  height: 100vh;
+  width: 100vw;
+  background-image: url(./components/images/start-img.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+h2 {
+  color: white;
+  font-size: 35px;
+}
+#nameInput {
+  height: 50px;
+  width: 12rem;
+  margin: 10px;
+  text-align: center;
+  font-size: large;
+  font-weight: 500;
+  background: rgba(32, 31, 31, 0.863);
+  color: white;
+  border-radius: 10px;
+  border: 3px solid white;
+}
+
+.form-container {
+  display: flex;
+  flex-direction: column;
+  z-index: 1;
+  color: white;
+  text-align: justify;
+}
+.first-que {
+  margin: 20px;
+  font-weight: 500;
+  font-size: 20px;
+  letter-spacing: 1px;
+  text-align: justify;
+}
+.container {
+  width: 500px;
+  background: rgba(32, 31, 31, 0.5);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+.select {
+  border-radius: 10px;
+  border: 3px solid white;
+  margin-left: 30px;
+  height: 50px;
+  font-size: 20px;
+
+  background: rgba(32, 31, 31, 0.1);
+  color: white;
+}
+.button {
+  margin-top: 40px;
+  margin-bottom: 40px;
+  text-align: center;
+  border-radius: 10px;
+  padding: 5px;
+  width: 200px;
+  color: white;
+  background: radial-gradient(
+    circle,
+    rgba(23, 22, 49, 0.863) 0%,
+    rgba(149, 149, 247, 0.253) 35%,
+    rgb(34, 37, 37) 100%
+  );
+}
+.button:hover {
+  cursor: pointer;
+}
+
+.header {
+  text-align: center;
+  color: white;
+  height: 100vh;
+  width: 100vw;
+  background-image: url(./components/images/start-img.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.intro {
+  z-index: 1;
+  color: white;
+  font-size: 20px;
+  margin-top: 30px;
+  z-index: 2;
+}
+.summary {
+  text-align: center;
+  color: rgba(255, 255, 255, 0.925);
+  height: 100vh;
+  width: 100vw;
+  background-image: url(./components/images/end-img.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  z-index: 1;
+  font-size: 20px;
+}
```

Comment:
Small comment is to merge divs with same styling to make the code even shorter. For example you use same styling in both intro and summary, you could maybe write like this:

.intro, .summary{
z-index: 1;
font-size: 20px;
}